docs/deployment.md

```diff
@@ -1,18 +0,0 @@
-# Deployment
-
-We don't deploy a docker image with Lostrómos, but instead only release the binary to github so that it can be built in
-to the image you need for your particular application. This is due to the link between kubectl and Lostromos, and it's
-possible the kubectl we intend to use isn't valid for everyones use case. We do build a docker image as part of testing
-that can be thought of as an [example](../test/docker/Dockerfile) however.
-
-## GOOS Environment Variable
```

Comment:
Where does this end up getting moved to? Is it cut out now?

Reply:
Yes. I cut some stuff for clarity with the idea of circling back.